### PR TITLE
[[Docs]] to - use variable naming convention

### PR DESCRIPTION
**Pull request by seaniepie**
- firstPoint and secondPoint changed to myFirstPoint and mySecondPoint to make it clearer they are variables
- isRaining changed to cIsRaining to follow custom property naming convention

### CODE REVIEW

**Comment by seaniepie** [comment]
Is there anything wrong with this one? It seems to have been missed.

File: docs/dictionary/keyword/to.lcdoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ OS: mac, windows, linux, ios, android
 Platforms: desktop, server, mobile
 
 Example:
-drag button 5 from firstPoint to secondPoint with shiftKey
+drag button 5 from myFirstPoint to mySecondPoint with shiftKey
```
Comment (montegoulding) [inline docs/dictionary/keyword/to.lcdoc]: Hmm... If we are following the naming convention they should be `tFirstPoint` etc

Comment (seaniepie) [reply docs/dictionary/keyword/to.lcdoc]: 👍 Sounds good to me

**Comment by peter-b** [comment]
@seaniepie Sorry this one's taken a long time to get looked at -- it slipped under the radar somehow.

@livecode-vulcan review ok e7a147c

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok e7a147c1dcee7da7a1934392ae24b45d869e13f6

**Comment by livecode-vulcan** [comment]
😎 test success e7a147c1dcee7da7a1934392ae24b45d869e13f6

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/649)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/654)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/649)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/653)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/658)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/650)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/650)

**Comment by seaniepie** [comment]
@peter-b , no worries about the time. I'm working on a number of tv ads and BBC Watchdog items at the moment so have not had the time to do more coding. Should be back into it in a couple of weeks. Thanks for the review.

**Pull request merged**